src/GPUActivityAgent.hpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2015, 2016, 2017, 2018, 2019, Intel Corporation
```

Comment:
```suggestion
 * Copyright (c) 2015 - 2022, Intel Corporation
```

Please make this change everywhere that it's needed in this PR.